types/jexcel/index.d.ts

```diff
@@ -0,0 +1,895 @@
+    thead: HTMLTableHeaderCellElement;
+    toolbar: HTMLDivElement;
+    undo: () => any;
+    up: (shiftKey: any, ctrlKey: any) => any;
+    updateCell: (x: number, y: number, value?: CellValue, force?: any) => any;
+    updateCopySelection: (x3: any, y3: any) => any;
+    updateCornerPosition: () => any;
+    updateFormula: (formula: any, referencesToUpdate: any) => any;
+    updateFormulaChain: (x: any, y: any, records: any) => any;
+    updateFormulas: (referencesToUpdate: any) => any;
+    updateMeta: (affectedCells: any) => any;
+    updateOrder: (rows: any) => any;
+    updateOrderArrow: (column: any, order: any) => any;
+    updatePagination: () => any;
+    updateScroll: (direction: any) => any;
+    updateSelection: (el1: any, el2: any, origin: any) => any;
+    updateSelectionFromCoords: (x1: number, y1: number, x2: number, y2: number, origin: any) => any;
+    updateTable: () => any;
+    updateTableReferences: () => any;
+    whichPage: (cell: any) => any;
+}
+
+interface JExcel {
+    (element: HTMLDivElement, options?: Options): JExcelElement;
+
+    //
+    // Other helpers
+    // ----------------------------------------------------------------------
+    build: any;
+    // tslint:disable-next-line ban-types
+    contextMenuControls: Function;
+    // tslint:disable-next-line ban-types
+    copyControls: Function;
+    createTabs: (element: HTMLDivElement, result: any[]) => void;
+    current: any;
+    // tslint:disable-next-line ban-types
+    cutControls: Function;
+    // tslint:disable-next-line ban-types
+    destroy: Function;
+    // tslint:disable-next-line ban-types
+    doubleClickControls: Function;
+    // tslint:disable-next-line ban-types
+    doubleDigitFormat: Function;
+    fromSpreadsheet: (result: any[]) => void;
+    // tslint:disable-next-line ban-types
+    getColumnName: Function;
+    // tslint:disable-next-line ban-types
+    getColumnNameFromId: Function;
+    // tslint:disable-next-line ban-types
+    getElement: Function;
+    // tslint:disable-next-line ban-types
+    getIdFromColumnName: Function;
+    // tslint:disable-next-line ban-types
+    injectArray: Function;
+    isMouseAction: boolean;
+    // tslint:disable-next-line ban-types
+    keyDownControls: Function;
+    methods: object;
+    // tslint:disable-next-line ban-types
+    mouseDownControls: Function;
+    // tslint:disable-next-line ban-types
+    mouseMoveControls: Function;
+    // tslint:disable-next-line ban-types
+    mouseOverControls: Function;
+    // tslint:disable-next-line ban-types
+    mouseUpControls: Function;
+    // tslint:disable-next-line ban-types
+    pasteControls: Function;
+    timeControl: any;
+    timeControlLoading: any;
+    // tslint:disable-next-line ban-types
+    touchEndControls: Function;
+    // tslint:disable-next-line ban-types
+    touchStartControls: Function;
+    // tslint:disable-next-line ban-types
+    validLetter: Function;
+}
+
+declare const jexcel: JExcel;
+export default jexcel;
```

Comment:
I didn't saw a default export. But instead I did see on the code that this library looks like it uses UMD. If that's the case you need to use an `export as namespace` syntax.

See: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html for a sample template.

Reply:
@armanio123 
Thank you for the review!
Until now, I didn't understand UMD well.

I correct the definition as UMD.
Could you check the src again?